Makefile

```diff
@@ -1,6 +1,26 @@
 # Override this to install docs somewhere else
 DOCDIR = /usr/share/doc/packages
 
+OS=$(shell egrep ^ID= /etc/os-release | sed -e 's/ID="\(.*\)"/\1/g')
```

Comment:
@rjfd I always wonder, when confronted with code that parses `/etc/os-release` using sed, awk, etc., whether it would be possible to eliminate that code by simply sourcing `/etc/os-release` as was originally intended. For example

```
OS=$(source /etc/os-release ; echo $ID)
```

Reply:
Fixed!